ui/app/AppLayouts/Wallet/panels/ManageAssetsPanel.qml

```diff
@@ -138,18 +109,18 @@ Control {
             model: root.controller.communityTokenGroupsModel
             implicitHeight: contentHeight
             interactive: false
-            spacing: Style.current.halfPadding
 
-            displaced: Transition {
-                NumberAnimation { properties: "x,y"; easing.type: Easing.OutQuad }
-            }
+            displaced: DisplacedTransition {}
 
             delegate: ManageTokensGroupDelegate {
                 controller: root.controller
                 dragParent: root
                 dragEnabled: root.controller.communityTokenGroupsModel.count > 1
-                communityGroupsExpanded: d.communityGroupsExpanded
             }
         }
     }
+
+    component DisplacedTransition: Transition {
+        NumberAnimation { properties: "x,y"; easing.type: Easing.OutQuad }
+    }
```

Comment:
Causes crash of tests runner on 5.15.2 :(

Reply:
On it